doc/integrator/ngeo_new_interface.rst

```diff
@@ -36,35 +36,50 @@ Then, if you want a new mobile interface, get the default files as follows:
 
 .. prompt:: bash
 
-  cp CONST_create_template/<package>/templates/mobile.html <package>/templates/<inferface>.html
-  cp CONST_create_template/<package>/static-ngeo/less/mobile.less <package>/templates/<inferface>.less
-  cp CONST_create_template/<package>/static-ngeo/js/mobile.js <package>/static-ngeo/js/<inferface>.js
+  cp CONST_create_template/<package>/templates/mobile.html <package>/templates/<interface>.html
+  cp CONST_create_template/<package>/static-ngeo/less/mobile.less <package>/static-ngeo/less/<interface>.less
+  cp CONST_create_template/<package>/static-ngeo/js/mobile.js <package>/static-ngeo/js/<interface>.js
 
 If you want a new desktop interface, get the default files as follows:
 
 .. prompt:: bash
 
-  cp CONST_create_template/<package>/templates/desktop.html <package>/templates/<inferface>.html
-  cp CONST_create_template/<package>/static-ngeo/less/desktop.less <package>/templates/<inferface>.less
-  cp CONST_create_template/<package>/static-ngeo/js/desktop.js <package>/static-ngeo/js/<inferface>.js
+  cp CONST_create_template/<package>/templates/desktop.html <package>/templates/<interface>.html
+  cp CONST_create_template/<package>/static-ngeo/less/desktop.less <package>/static-ngeo/less/<interface>.less
+  cp CONST_create_template/<package>/static-ngeo/js/desktop.js <package>/static-ngeo/js/<interface>.js
 
 Edit interface files
 ~~~~~~~~~~~~~~~~~~~~
-In the file ``<package>/static-ngeo/js/<inferface>.js``, adapt the ``goog.provide`` statement to the
-name of the new interface. For example, if your package is called "demo" and your new interface is
-called "new", you need to change ``goog.provide('demo_desktop');`` to ``goog.provide('demo_new');``
+In the file ``<package>/static-ngeo/js/<interface>.js``, adapt the following lines:
 
-In the file ``<package/static-ngeo/templates/<interface>.html``, adapt the name of the referenced css
-and js files to the new interface name. For example, if your package is called "demo" and your new
-interface is called "new", you need to change the file as follows:
+.. code:: js
+
+   * This file defines the "<package>_<interface>" Closure namespace, which is be used as
+   ...
+   goog.provide('<package>.<Interface>Controller');
+   goog.provide('<package>_<interface>');
+   ...
+   geoportal.<Interface>Controller = function($scope, $injector) {
+   ...
+   ol.inherits(<package>.<Interface>Controller, gmf.AbstractDesktopController);
+   ...
+   geoportal.module.controller('<Interface>Controller', <package>.<Interface>Controller);
+
+where you replace ``<package>`` with the name of your package and ``<interface>`` with the name of your new
+interface.
```

Comment:
For me this sentences steel isn't clear for me it should be:
```
where ``<package>`` is the name of your package, you should replace ``<interface>`` (desktop in the original file) with the name of your new interface.
```

Reply:
sentence changed